remote/remote_state_s3_test.go

```diff
@@ -276,6 +276,7 @@ func TestGetTerraformInitArgs(t *testing.T) {
 				"skip_bucket_ssencryption":       false,
 				"skip_bucket_root_access":        false,
 				"skip_bucket_enforced_tls":       false,
+				"disable_bucket_update":          false,
```

Comment:
I think will be good to have a test for case `disable_bucket_update=true`

Reply:
changed